frontend/packages/noobaa-storage-plugin/integration-tests/views/obcPage.view.ts

```diff
@@ -21,30 +22,67 @@ export const goToOBPage = async () => {
 const createOBC = element(by.buttonText('Create Object Bucket Claim'));
 
 const obcNameInput = $('#obc-name');
-const scDropdown = $('#sc-dropdown');
-const bcDropdown = $('#bc-dropdown');
-const obcSC = $('#openshift-storage\\.noobaa\\.io-link');
-const defaultBC = $('a#noobaa-default-bucket-class-link');
+// doesn't work in OCP 4.3
+// const scDropdown = $('#sc-dropdown');
+const scDropdown = $('div.co-storage-class-dropdown div button');
+// const bcDropdown = $('#bc-dropdown');
+const bcDropdown = $('div.nb-create-obc__bc-dropdown div button');
+// const defaultBC = $('a#noobaa-default-bucket-class-link');
 const createBtn = element(by.buttonText('Create'));
 
 // Resource Page
 const resourceName = $('.co-resource-item__resource-name');
 const resourceStatus = $('.co-resource-item__resource-status');
 const showValues = element(by.partialButtonText('Reveal Values'));
+const hideValues = element(by.partialButtonText('Hide Values'));
+const endpoint = element(by.xpath('//dt[text()="Endpoint"]/following-sibling::dd/div/pre'));
+const accessKey = element(by.xpath('//dt[text()="Access Key"]/following-sibling::dd/div/pre'));
+const secretKey = element(by.xpath('//dt[text()="Secret Key"]/following-sibling::dd/div/pre'));
+export const obcName = element(by.xpath('//dl/dt[text()="Name"]/following-sibling::dd'));
+const obcNamespace = element(by.xpath('//dl/dt[text()="Namespace"]/following-sibling::dd/span/a'));
+const obcLabels = element(by.xpath('//dl/dt[text()="Labels"]/following-sibling::dd'));
+const obcAnnotations = element(by.xpath('//dl/dt[text()="Annotations"]/following-sibling::dd'));
+const obcCreationDate = element(by.xpath('//dt[text()="Created At"]/following-sibling::dd'));
+const obcOwner = element(by.xpath('//dt[text()="Owner"]/following-sibling::dd'));
+const obcSecret = element(by.xpath('//dt[text()="Secret"]/following-sibling::dd/span/a'));
+const obcStatus = element(by.xpath('//dt[text()="Status"]/following-sibling::dd'));
+const obcStorageClass = element(
+  by.xpath('//dt[text()="Storage Class"]/following-sibling::dd/span/a'),
+);
+export const obcObjectBucket = element(
+  by.xpath('//dt[text()="Object Bucket"]/following-sibling::dd/span/a'),
+);
```

Comment:
`xpath` should always be our last option. Can we maybe put `data-testid` in the fields where required?

Reply:
Our problem is, we need to run these tests on OCP 4.3 version (and at this moment it's our priority), so I can't add any data-test-id attributes or anything else to the code of console itself.